modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import {ClayPaginationWithBasicItems} from '@clayui/pagination';
+import React, {useContext, useEffect, useState} from 'react';
+import {Link} from 'react-router-dom';
+
+import {AppContext} from '../../AppContext.es';
+import KeywordsList from '../../components/KeywordList.es';
+import UserAvatar from '../../components/UserAvatar.es';
+import {getThreads} from '../../utils/client.es';
+import {dateToInternationalHuman} from '../../utils/utils.es';
+
+export default () => {
+	const context = useContext(AppContext);
+
+	const [loading, setLoading] = useState(true);
+	const [page, setPage] = useState(1);
+	const [pageSize] = useState(5);
+	const [questions, setQuestions] = useState([]);
+
+	useEffect(() => {
+		getThreads({page, pageSize, siteKey: context.siteKey})
+			.then(data => setQuestions(data))
+			.then(() => setLoading(false));
+	}, [page, pageSize, context.siteKey]);
+
+	const hasValidAnswer = question =>
+		question.messageBoardMessages.items.filter(
+			message => message.showAsAnswer
+		).length > 0;
+
+	return (
+		<section>
+			{loading && <ClayLoadingIndicator/>}
+			{questions.items &&
+			 questions.items.map(question => (
+				 <div key={question.id}>
+
+					 <div className="autofit-row autofit-padded">
+						 <div className="autofit-col autofit-col-expand">
+							 <div className="autofit-section">
+								 <h2>
+									 <Link to={'/questions/' + question.id}>
+										 {question.headline}
+									 </Link>
+								 </h2>
+							 </div>
+						 </div>
+						 <div className="autofit-col">
+							 <p>
+								 <small>
```

Comment:
this markup is strange

Reply:
Fixed here: https://github.com/boton/liferay-portal/pull/122/commits/8e1d3ec75bd91ba8193632afbbbaae3bb1818b6c